tests/jerry/array-prototype-sort.js

```diff
@@ -43,9 +43,20 @@ function f(arg1, arg2) {
 	}
 }
 
-var sorted = array.sort(f);
+array.sort(f);
 for (i = 0; i < array.length; i++) {
-	assert(sorted[sorted.length - i - 1] === i);
+	assert(array[array.length - i - 1] === i);
+}
+
+// Sorting sparse array
+var array = [1,,2,,3,,4,undefined,5];
+var expected = [1,2,3,4,5,,,,];
```

Comment:
I think, this is because values are copied to simple array of values without additional information.

It seems that the way also can lead to out-of-memory in case of sparse arrays. For example:
```
a = new Array ();
a[100000] = 1;
a[1000000] = 2;
a.sort ();
```

Is it correct?